src/goto-programs/goto_trace.cpp

```diff
@@ -242,6 +242,15 @@ std::string trace_numeric_value(
       }
     }
   }
+  else if(expr.id() == ID_annotated_pointer_constant)
+  {
+    const auto &annotated_pointer_constant =
+      to_annotated_pointer_constant_expr(expr);
+    auto width = to_pointer_type(expr.type()).get_width();
+    auto value = annotated_pointer_constant.get_value();
```

Comment:
Nit pick: I'd use `const auto &` here, for that's what we usually do for `irep_idt`. Arguably, the copy will be no more expensive in case of `dstringt`, perhaps even cheaper. With `USE_STRING` the situation might be rather different.

Reply:
ok